tests/cache_tests.py

```diff
@@ -68,6 +70,31 @@ def test_memcache_keyprefix(self):
             'Memcache returned a value even though it should have been empty')
 
 
+    def test_config(self):
```

Comment:
> I would create a new test file. Because we do not intent to test cache, we just intent to test Configuration object creation.

yup, that's what I have in my local commit, I just wanted to try it out first